Reorganize `astro workspace` docs by breaking out subcommands into separate docs (#1978)
astronomer/docs: astro/cli/astro-workspace-user-add.md

---
sidebar_label: "astro workspace user add"
title: "astro workspace user add"
id: astro-workspace-user-add
description: Reference documentation for astro workspace.
hide_table_of_contents: true
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::info  

> The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts. 

Reviewer: ```suggestion
The behavior and format of this command differs depending on which Astronomer product you're using. Use the following tabs to change product contexts. 
```

:::

<Tabs
    defaultValue="astro"
    values={[
        {label: 'Astro', value: 'astro'},
        {label: 'Software', value: 'software'},
    ]}>
<TabItem value="astro">

> Invite an existing user to your current Astronomer Workspace.

Reviewer: ```suggestion
Add an existing Organization user to your current Astronomer Workspace as an Admin, Editor, or Viewer. You must be a Workspace Admin to perform this action. After you add the user, their information appears in the Access tab as a new entry in the Members list. To access the Workspace, the user needs to accept the invitation sent by email and log in.

This command is functionally equivalent to using the **Add member** button in the Workspace Settings view of the Cloud UI.
```

Maybe too much? Feel free to cut me here

## Usage

```sh
astro workspace user add <email> 
```

## Options

| Option    | Description                                          | Valid Values                                                                               |
| --------- | ---------------------------------------------------- | ------------------------------------------------------------------------------------------ |

> | `<email>` | The email for the user you want to invite or update. | Any valid email                                                                            |

Reviewer: ```suggestion
| `<email-address>` | The email address of the user that you want to add to the Workspace. | Any valid email address                                                                            |
```

| `--email` (_Required_) | The user's email                                                                                                                                      | Any valid email address                                                                                                 |
| `--workspace-id`       | The Workspace that the user is added to. Specify this flag if you want to create a user in a Workspace that is different than your current Workspace. | Any valid Workspace ID                                                                                                  |
| `--role`               | The role assigned to the user.                                                                                                                        | Possible values are `WORKSPACE_VIEWER`, `WORKSPACE_EDITOR`, and `WORKSPACE_ADMIN`. Default value is `WORKSPACE_VIEWER`. |

## Related documentation

- [Manage Workspaces and Deployments on Astronomer](https://docs.astronomer.io/software/manage-workspaces)

> - [Manage User Permissions on Astronomer](https://docs.astronomer.io/software/workspace-permissions)

Reviewer: Add https://docs.astronomer.io/astro/add-user?

---
sidebar_label: "astro workspace user add"
title: "astro workspace user add"
id: astro-workspace-user-add
description: Reference documentation for astro workspace.
hide_table_of_contents: true
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::info  

The behavior and format of this command differs depending on which Astronomer product you're using. Use the following tabs to change product contexts. 

:::

<Tabs
    defaultValue="astro"
    values={[
        {label: 'Astro', value: 'astro'},
        {label: 'Software', value: 'software'},
    ]}>
<TabItem value="astro">

> Add an existing Organization user to your current Astronomer Workspace. You must be a Workspace Owner to perform this action.
> 
> You can use this command to invite multiple users to a Workspace at a time. See [Add a group of users to Astro using the Astro CLI](add-user.md#add-a-group-of-users-to-astro-using-the-astro-cli)
> 

Author: ```suggestion
Add an existing Organization user to your current Astronomer Workspace. You must be a Workspace Admin to perform this action.

```
